Form/Type/CategorySelectorType.php

```diff
@@ -65,7 +65,7 @@ public function configureOptions(OptionsResolver $resolver)
                 'context' => null,
                 'category' => null,
                 'choice_list' => function (Options $opts, $previousValue) use ($that) {
-                    return new SimpleChoiceList($that->getChoices($opts));
+                    return new ArrayChoiceList($that->getChoices($opts));
```

Comment:
This code will never be executed on SF 3.x
`choice_list` option will be used only on <SF 2.7